js/presentation.js

```diff
@@ -4,6 +4,11 @@ import Slide from "components/Slide.react";
 import Code from "components/Code.react";
 import DocumentTitle from "react-document-title";
 
+// Prism JS configuration
+import 'prismjs/components/prism-json';
+import 'prismjs/components/prism-jsx';
+import "style!css!prismjs/themes/prism.css"; // Theme
```

Comment:
I even think that languages should be required dynamically by the `Code` component if necessary. Theme could be a in the config.